src/ralph/assets/models/choices.py

```diff
@@ -65,3 +65,15 @@ class ComponentType(Choices):
     backup = _('backup')
     software = _('software')
     os = _('operating system')
+
+
+class EthernetSpeed(Choices):
+    _ = Choices.Choice
+
+    s10mbit = _('10 Mbps')
+    s100mbit = _('100 Mbps')
+    s1gbit = _('1 Gbps')
+    s10gbit = _('10 Gbps')
```

Comment:
what about 40gbps and 100gbps? https://en.wikipedia.org/wiki/100_Gigabit_Ethernet